app/com.zoop.zoopPC/com.zoop.zoopPc.yml

```diff
@@ -0,0 +1,21 @@
+app-id: com.zoop.zoopPc
```

Comment:
Is there a website at zoop.com ? Because I can't open it on my browser

Reply:
As our domain is Brazilian, we are analyzing your suggestions and considering the best approach for our needs. Thank you for your assistance.

https://www.zoop.com.br/

Reply:
there is no mystery, it should be `br.com.zoop`. Like documentation says.